apis/v1beta1/types.go

```diff
@@ -21,6 +21,17 @@ type ProviderConfigSpec struct {
 	// of AWS calls made by the provider.
 	// +optional
 	Endpoint *EndpointConfig `json:"endpoint,omitempty"`
+	// Whether to skip credentials validation via the STS API.
+	// This can be useful for testing and for AWS API implementations that do not have STS available.
+	SkipCredsValidation bool `json:"skip_credentials_validation,omitempty"`
```

Comment:
I think this is an optional field. Could you please the `+optional` mark for this?